account-keys/src/account_keys.rs

```diff
@@ -171,6 +175,15 @@ impl PublicAddress {
     }
 }
 
+impl AuthorityVerifier for PublicAddress {
+    type Sig = <RistrettoPublic as AuthorityVerifier>::Sig;
+    type Error = <RistrettoPublic as AuthorityVerifier>::Error;
+
+    fn verify_authority(&self, spki_bytes: &[u8], sig: &Self::Sig) -> Result<(), Self::Error> {
```

Comment:
why does this take a sig as a parameter? shouldn't it take the sig from Self ?

Reply:
Under all the covers, the trait is implemented primarily on the `RistrettoPublic` type, so the trait needs to have the sig passed in.